artemis-server/src/main/java/org/apache/activemq/artemis/core/security/impl/SecurityStoreImpl.java

```diff
@@ -101,15 +103,15 @@ public SecurityStoreImpl(final HierarchicalRepository<Set<Role>> securityReposit
       if (authenticationCacheSize == 0) {
          authenticationCache = null;
       } else {
-         authenticationCache = CacheBuilder.newBuilder()
+         authenticationCache = Caffeine.newBuilder()
                                            .maximumSize(authenticationCacheSize)
                                            .expireAfterWrite(invalidationInterval, TimeUnit.MILLISECONDS)
                                            .build();
```

Comment:
This is a bit of a nit-pick, but the lines should be re-aligned so all the `.` characters are in line vertically.

Reply:
Fair. That was probably result of rebasing the changes again on top of your earlier PR, and those lines not having changes on them.